Library/include/CSP/Systems/Users/Authentication.h

```diff
@@ -188,95 +153,16 @@ class CSP_API AgoraUserTokenParams
 {
 public:
 	csp::common::String AgoraUserId;
-	int Lifespan;
 	csp::common::String ChannelName;
+	int Lifespan;
```

Comment:
This member was moved to reduce the size of this type by 8 bytes due to padding not being needed.